instructor/templates/instructor/assignment_sidebar.html

```diff
@@ -9,7 +9,7 @@
             <a class='scb_s_assignment_sidebar_link'
                href="{% if new %}{% url "common_assignment_setup" %}
                {% else %}{% url "common_assignment_modify" %}{% endif %}">
-                 Assignment setup
+                 Assignment
```

Comment:
It looks like #394 was to change the heading and leave the subheading alone, but this changes the subheading and leaves the heading alone